desktop/src/backends/ui.rs

```diff
@@ -429,7 +437,9 @@ fn load_fontdb_font(name: String, face: &FaceInfo) -> Result<FontDefinition<'sta
                 name,
                 is_bold,
                 is_italic,
-                data: bin.as_ref().as_ref().to_vec(),
+                // TODO remove when https://github.com/rust-lang/rust-clippy/issues/15252 is fixed
+                #[allow(clippy::unnecessary_to_owned)]
+                data: FontFileData::new(bin.as_ref().as_ref().to_vec()),
```

Comment:
Couldn't this simply do `FontFileData(bin)`? (assuming we make the `FontFileData` field public) It would nicely avoid a copy, too.